applications/polkadot-mempool-explorer-v2.md

```diff
@@ -0,0 +1,499 @@
+| ------: | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **0a.** | License                                | APACHE 2                                                                                                                                                                                                                     |
+| **0b.** | Documentation                          | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how to view pending transactions in dashboard |
+| **0c.** | Testing and Testing Guide              | Core functions will be fully covered by comprehensive unit tests to ensure functionality and robustness. In the guide, we will describe how to run these tests.                                                              |
+| **0d.** | Docker                                 | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.                                                                                                                |  |  |  |
+|      1. | new UI for desktop and mobile view and fix existing issues|                                                                                                                                            |
+|      2. | enhance the APIs and fix existing issues                                                                                                                                                                        |
+|      3. | enable CI/CD | spin up a server instance for hosting the backend and deploying the frontend at github pages                                                                                                                                                     |
+  | 4a | polkadot js API tutorials             | We will provide both **inline documentation** of the code and a basic **tutorial** that explains how to run sample examples |
+| 4b | Graphics/Illustrations | Illustrations wherever required to explain complex topics
+
+
+
+
+Chapters overview
+
+  
+
+1. Explain the basics of polkadot architecture
+
+2. Role of polkadot JS API, substrate and how to interact with the live blockchain
+
+3. How to fetch the metadata, and what are the metadata of the blockchain? https://polkadot.js.org/docs/api/start/basics
+
+4. What's the purpose of polkadot js console and how to use it? https://polkadot.js.org/apps/#/
+
+5. Explain transaction lifecycle, and how to perform transactions through the API
+
+and more .....
+
+Overall goal it to curate a series of tutorials to build the concepts of polkadot blockchain.
```

Comment:
You mention a "series of tutorials" here, but the deliverables only mention "a basic tutorial". Can you clarify this?

Reply:
@semuelle fixed this. there was a minor typo. It is supposed to be series of tutorials.